src/main/java/com/courtcanva/ccfranchise/services/OrderService.java

```diff
@@ -22,23 +23,22 @@ public class OrderService {
     private final OrderRepository orderRepository;
     private final OrderMapper orderMapper;
 
+    public List<OrderPendingPostDto> getFirstTenOpenOrdersById(Long franchiseeId) {
```

Comment:
Better to make sure we name out entities consistently, pick either `PendingOrders` or `OpenOrders`